docs/quickstart.md

```diff
@@ -0,0 +1,154 @@
+---
+post_title: Spark Quickstart
+menu_order: 10
+feature_maturity: stable
+enterprise: 'no'
+---
+
+This tutorial will get you up and running in minutes with Spark. You will install the DC/OS Spark.
+
+**Prerequisites:**
+
+-  [DC/OS and DC/OS CLI installed](https://docs.mesosphere.com/1.9/installing/) with a minimum of three agent nodes with eight GB of memory and ten GB of disk available on each agent.
+-  Depending on your [security mode](https://docs.mesosphere.com/1.9/overview/security/security-modes/), Spark requires a service authentication for access to DC/OS. For more information, see [Configuring DC/OS Access for Spark](https://docs.mesosphere.com/service-docs/spark/spark-auth/).
+
+   | Security mode | Service Account |
+   |---------------|-----------------------|
+   | Disabled      | Not available   |
+   | Permissive    | Optional   |
+   | Strict        | Required |
+
+
+1.  Install the Spark package. This may take a few minutes.
+
+    ```bash
+    dcos package install spark
+    ```
+    
+    Your output should resemble:
+    
+    ```bash
+    Installing Marathon app for package [spark] version [1.1.0-2.1.1]
+    Installing CLI subcommand for package [spark] version [1.1.0-2.1.1]
+    New command available: dcos spark
+    DC/OS Spark is being installed!
+    
+    	Documentation: https://docs.mesosphere.com/service-docs/spark/
+    	Issues: https://docs.mesosphere.com/support/
+    ```
+   
+    **Tips:** 
+    
+    -  You can view the status of your Spark installation from the DC/OS GUI **Services** tab.
+       
+       ![Verify spark installation](/img/spark-gui-install.png)
+       
+    -  Type `dcos spark` to view the Spark CLI options.
+
+1.  Run the sample Java Spark Pi job for DC/OS. This runs a Spark job which calculates the value of Pi. You can view the example source [here](https://downloads.mesosphere.com/spark/assets/spark-examples_2.10-1.4.0-SNAPSHOT.jar). 
+
+    1.  Run this command: 
+
+        ```bash
+        dcos spark run --submit-args="--class org.apache.spark.examples.SparkPi https://downloads.mesosphere.com/spark/assets/spark-examples_2.10-1.4.0-SNAPSHOT.jar 30"
```

Comment:
The DC/OS version is irrelevant.  It's the Spark version and the Jar version that must be compatible.  Any Spark version >= 2.0 will not work with a 2.10 jar